### PR TITLE
Add `database_routing` to resource kusto eventhub

### PR DESCRIPTION
**Pull request by liuwuliuyun**
Add `database_routing` property to resource kusto eventhub data connection

### CODE REVIEW

File: website/docs/r/kusto_eventhub_data_connection.html.markdown [hunk 1]
```diff
@@ -107,6 +106,8 @@ The following arguments are supported:
 
 * `data_format` - (Optional) Specifies the data format of the EventHub messages. Allowed values: `APACHEAVRO`, `AVRO`, `CSV`, `JSON`, `MULTIJSON`, `ORC`, `PARQUET`, `PSV`, `RAW`, `SCSV`, `SINGLEJSON`, `SOHSV`, `TSVE`, `TSV`, `TXT`, and `W3CLOGFILE`.
 
+* `database_routing` - (Optional) Indication for database routing information from the data connection, by default only database routing information is allowed. Allowed values: `Single`, `Multi`.
```
Comment (katbyte) [inline website/docs/r/kusto_eventhub_data_connection.html.markdown:109]: should we make this
```suggestion
* `database_routing_type` - (Optional) Indication for database routing information from the data connection, by default only database routing information is allowed. Allowed values: `Single`, `Multi`.
```
?

**Review by katbyte — CHANGES_REQUESTED** [review]

File: website/docs/r/kusto_eventhub_data_connection.html.markdown (hunk 1 above)
Comment (liuwuliuyun) [reply website/docs/r/kusto_eventhub_data_connection.html.markdown:109]: Ok, let me change this to `database_routing_type`

Comment (liuwuliuyun) [reply website/docs/r/kusto_eventhub_data_connection.html.markdown:109]: Shall we change all `database_routing` in other two PRs to `database_routing_type`? Although in azure API, this was called databaseRouting.

**Comment by liuwuliuyun** [comment]
Test after change:

GOROOT=D:\Program Files\Go #gosetup
GOPATH=C:\Users\yunliu1\go #gosetup
"D:\Program Files\Go\bin\go.exe" test -c -o C:\Users\yunliu1\AppData\Local\Temp\GoLand\___TestAccKustoEventHubDataConnection_databaseRoutingType_in_github_com_hashicorp_terraform_provider_azurerm_internal_services_kusto.test.exe github.com/hashicorp/terraform-provider-azurerm/internal/services/kusto #gosetup
"D:\Program Files\Go\bin\go.exe" tool test2json -t C:\Users\yunliu1\AppData\Local\Temp\GoLand\___TestAccKustoEventHubDataConnection_databaseRoutingType_in_github_com_hashicorp_terraform_provider_azurerm_internal_services_kusto.test.exe -test.v -test.paniconexit0 -test.run ^\QTestAccKustoEventHubDataConnection_databaseRoutingType\E$ #gosetup
=== RUN   TestAccKustoEventHubDataConnection_databaseRoutingType
=== PAUSE TestAccKustoEventHubDataConnection_databaseRoutingType
=== CONT  TestAccKustoEventHubDataConnection_databaseRoutingType
--- PASS: TestAccKustoEventHubDataConnection_databaseRoutingType (1388.39s)
PASS


Process finished with the exit code 0

**Review by katbyte — APPROVED** [review]
Thanks @liuwuliuyun - LGTM 🌵

**Pull request merged**